search/api.py

```diff
@@ -162,7 +167,15 @@ def search_percolate_queries(program_enrollment_id):
         django.db.models.query.QuerySet: A QuerySet of PercolateQuery matching the percolate results
     """
     conn = get_conn()
-    result = conn.percolate(get_default_alias(), USER_DOC_TYPE, id=program_enrollment_id)
+    enrollment = ProgramEnrollment.objects.get(id=program_enrollment_id)
+    doc = serialize_program_enrolled_user(enrollment)
```

Comment:
Those tasks already have a lot of overhead since they are triggered when a single user's state changes. I'm not sure the extra `serialize_program_enrolled_user` call will change much. It won't slow down `recreate_index` because we don't trigger automatic emails for that case.

If it did become a bottleneck we could refactor `index_users` and `index_program_enrolled_users` to take the serialized dicts instead of the `ProgramEnrollment` objects, but I'm not sure what affect that would have on readability